README.md

```diff
@@ -75,31 +84,77 @@ Recovery settings configures Vlab recovery service support of `sshmux`. They are
 | `usernames` | `[]string` | Usernames dedicated to the recovery server.           | No       | `["recovery", "console"]` |
 | `token`     | `string`   | Token used to authenticate with the recovery backend. | No       | `"long-and-random-token"` |
 
-## API server
+## Auth API
+
+`sshmux` uses a RESTful API to perform authentication and authorization for a user.
+
+### `POST /v1/auth/:username`
+
+#### Input
+
+| Key               | Type                  | Description                                                                                    | Position | Optional |
```

Comment:
We're showing a "Required" column and an "Optional" column in the same document? Might be better to keep it consistent (I prefer "Required").

Reply:
I chose "optional" over "required" here because I didn't consider these as requirements for the implementer. Since "required" is already an established term from JSON schema and OpenAPI spec, there's no reason to avoid it👍